packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/PhysicalObjectsAccordion/CuratorialAssessmentsList.js

```diff
@@ -1,16 +1,19 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { compose } from 'redux'
-import { Button, Header, List, Modal } from 'semantic-ui-react'
+import { Button, List, Modal } from 'semantic-ui-react'
 
 import config from 'config'
 import { pathBuilder } from 'coreModules/form/higherOrderComponents'
 import { FormModal } from 'coreModules/form/components'
+import formParts from 'coreModules/form/components/parts'
 import { createModuleTranslate } from 'coreModules/i18n/components'
 import CuratorialAssessmentItem from './CuratorialAssessmentItem'
 import CreateCuratorialAssessment from './CuratorialAssessmentForm/Create'
 import FieldsForTest from './CuratorialAssessmentForm/Test'
 
+const { TranslatedHeaderWithHelpIcon } = formParts
```

Comment:
maybe you can import `TranslatedHeaderWithHelpIcon` directly from the file instead of from formParts? since it's not used in a form unit.